### PR TITLE
feature(List): Finish Lists; add Connected People

### PR DESCRIPTION
**Pull request by geolessel**

### CODE REVIEW

File: test/pco_api/query_test.exs [hunk 1]
```diff
@@ -30,8 +30,8 @@ defmodule PcoApi.QueryTest do
   end
 
   test "per_page allows piping" do
-    expected = [{"where[b]", "bb"}, {"per_page", "30"}]
-    assert Query.where(b: "bb") |> Query.per_page(30)
+    expected = [{"per_page", "30"}, {"where[b]", "bb"}]
+    assert (Query.where(b: "bb") |> Query.per_page(30)) == expected
```
Comment (credoci) [inline test/pco_api/query_test.exs]: Pipe chain should start with a raw value.

File: lib/pco_api/people/list/rule/condition.ex [hunk 2]
```diff
@@ -0,0 +1,9 @@
+defmodule PcoApi.People.List.Rule.Condition do
```
Comment (credoci) [inline lib/pco_api/people/list/rule/condition.ex]: Modules should have a @moduledoc tag.

File: lib/pco_api/people/connected_people.ex [hunk 3]
```diff
@@ -0,0 +1,8 @@
+defmodule PcoApi.People.ConnectedPeople do
```
Comment (credoci) [inline lib/pco_api/people/connected_people.ex]: Modules should have a @moduledoc tag.

File: test/pco_api/people/list/rule/condition_test.exs [hunk 4]
```diff
@@ -0,0 +1,55 @@
+defmodule PcoApi.People.List.Rule.ConditionTest do
```
Comment (credoci) [inline test/pco_api/people/list/rule/condition_test.exs]: Modules should have a @moduledoc tag.

File: lib/pco_api/people/list/rule/result.ex [hunk 5]
```diff
@@ -0,0 +1,16 @@
+defmodule PcoApi.People.List.Rule.Result do
```
Comment (credoci) [inline lib/pco_api/people/list/rule/result.ex]: Modules should have a @moduledoc tag.

File: test/pco_api/people/list/share_test.exs [hunk 6]
```diff
@@ -0,0 +1,64 @@
+defmodule PcoApi.People.List.ShareTest do
```
Comment (credoci) [inline test/pco_api/people/list/share_test.exs]: Modules should have a @moduledoc tag.

File: test/pco_api/people/connected_people_test.exs [hunk 7]
```diff
@@ -0,0 +1,48 @@
+defmodule PcoApi.People.ConnectedPeopleTest do
```
Comment (credoci) [inline test/pco_api/people/connected_people_test.exs]: Modules should have a @moduledoc tag.

File: test/pco_api/people/list/rule/condition/result_test.exs [hunk 8]
```diff
@@ -0,0 +1,55 @@
+defmodule PcoApi.People.List.Rule.Condition.ResultTest do
```
Comment (credoci) [inline test/pco_api/people/list/rule/condition/result_test.exs]: Modules should have a @moduledoc tag.

File: lib/pco_api/people/list/rule/condition/result.ex [hunk 9]
```diff
@@ -0,0 +1,9 @@
+defmodule PcoApi.People.List.Rule.Condition.Result do
```
Comment (credoci) [inline lib/pco_api/people/list/rule/condition/result.ex]: Modules should have a @moduledoc tag.

File: lib/pco_api/people/list/shares.ex [hunk 10]
```diff
@@ -0,0 +1,10 @@
+defmodule PcoApi.People.List.Share do
```
Comment (credoci) [inline lib/pco_api/people/list/shares.ex]: Modules should have a @moduledoc tag.

File: test/pco_api/query_test.exs [hunk 11]
```diff
@@ -46,6 +46,6 @@ defmodule PcoApi.QueryTest do
 
   test "after_record allows piping" do
     expected = [{"after", "100"}, {"per_page", "100"}]
-    assert Query.per_page(100) |> Query.after_record(100)
+    assert (Query.per_page(100) |> Query.after_record(100)) == expected
```
Comment (credoci) [inline test/pco_api/query_test.exs]: Pipe chain should start with a raw value.

File: test/pco_api/people/list/rule/result_test.exs [hunk 12]
```diff
@@ -0,0 +1,55 @@
+defmodule PcoApi.People.List.Rule.ResultTest do
```
Comment (credoci) [inline test/pco_api/people/list/rule/result_test.exs]: Modules should have a @moduledoc tag.

File: lib/pco_api/people/list.ex [hunk 13]
```diff
@@ -9,7 +9,6 @@ defmodule PcoApi.People.List do
 
   use PcoApi.Actions
 
-  def list, do: list([])
```
Comment (jessejanderson) [inline lib/pco_api/people/list.ex:12]: Why remove the no params list?

Comment (jessejanderson) [inline lib/pco_api/people/list.ex:12]: Oh, never mind - I see it's defined in Actions.List

**Comment by jessejanderson** [comment]
👍

**Pull request merged**